### ISSUE TITLE
Continue Shopping Button not working

### ISSUE THREAD

**Issue by tsarma** [isotope/core#1988]
https://github.com/isotope/core/blob/master/system/modules/isotope/library/Isotope/Frontend/ProductCollectionAction/ContinueShoppingAction.php#L30

Due to this condition check ```&& \Input::get('continue') != ''```. Even when I've checked ```Enable "Continue shopping" button``` in Isotope Cart Module, the button does not appear. If there is no specific reason for that condition please remove it.

**Comment by tsarma** [comment]
I think your indentation of ```Enable "Continue shopping" button```is something else, which I fail to understand. 
Had it worked the way I thought, then there should be a way to select a redirect page for the continue shopping button.

**Comment by aschempp** [comment]
The "Continue shopping" button only appears if the "continue link" is in the URL, which usually happens if you add a product to the cart.

**Comment by tsarma** [comment]
Thanks for explanation but for me the "continue link" does not appear when I add product to a cart. It may have appeared if I've defined a redirect to cart page from product list module ```Add product jump to page```, which is not in my case because I wanted the users to stay on product list page when he/she adds product to the cart.

Anyway I've solved in my situation by adding manually extra HTML code ```<a href="{{link_url::123}}" class="">Weiter einkaufen</a>``` in ```iso_collection_default``` template.  https://bit.ly/2S0SbKM
